src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollector.java

```diff
@@ -100,12 +100,24 @@ protected void addCorpCardPersonalExpenseDetailLine(ConcurStandardAccountingExtr
     protected Map<String, List<ConcurStandardAccountingExtractDetailLine>> getConsolidatedPersonalExpensesMapForLine(
             ConcurStandardAccountingExtractDetailLine detailLine) {
         if (detailLine.getJournalAmount().isPositive()) {
-            if (collectorHelper.lineRepresentsReturnOfCorporateCardPersonalExpenseToUser(detailLine)) {
-                return consolidatedCorpCardPersonalExpenseCredits;
-            } else {
-                return consolidatedCorpCardPersonalExpenseDebits;
-            }
-        } else if (collectorHelper.lineRepresentsReturnOfCorporateCardPersonalExpenseToUniversity(detailLine)) {
+            return getConsolidatedPersonalExpensesMapForDebitLine(detailLine);
+        } else {
+            return getConsolidatedPersonalExpensesMapForCreditLine(detailLine);
+        }
+    }
+
+    protected Map<String, List<ConcurStandardAccountingExtractDetailLine>> getConsolidatedPersonalExpensesMapForDebitLine(

```

Comment:
I think this is better, but I just want to ask if we want to have the function named ..ForDebitLine when it could return credits?